### PR TITLE
Fix #39, allow code blocks without trailing new line to run

### LINKED ISSUE #39: Code block without trailing new line can't be executed
**Describe the bug**

Code block without trailing new line can't be executed.
When I click CTRL + e, I see the error `Error: could not parse code block`.

**To Reproduce**

Add a code block to the end of the last slide without adding a trailing new line.

**Expected behavior**

Since Markdown doesn't have a problem to parse this kind of code block, I would expect the code block to be runnable.

### PR DESCRIPTION
**Pull request by suzaku**
Fix #39

### CODE REVIEW

File: internal/code/code.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ type Result struct {
 }
 
 // ?: means non-capture group
-var re = regexp.MustCompile("(?s)(?:```|~~~)(\\w+)\n(.*)\n(?:```|~~~)\n")
+var re = regexp.MustCompile("(?s)(?:```|~~~)(\\w+)\n(.*)\n(?:```|~~~)\\s?")
```
Comment (maaslalani) [inline internal/code/code.go:24]: This means that people can have trailing spaces or tabs after the three backticks? If it's optional anyway we can just remove the `\s?` and I believe everything should still work

Comment (suzaku) [reply internal/code/code.go:24]: I'm not sure, if we leave out the `\s`, would it match things like `~~~~~`?

Comment (maaslalani) [reply internal/code/code.go:24]: Oh yea you're totally right! My bad

**Review by maaslalani — APPROVED** [review]
Looks awesome, thanks for fixing this!

**Comment by suzaku** [comment]
Looking forward to use `slides` for a tech sharing next week 😉

**Comment by maaslalani** [comment]
Ahh thats amazing! So glad to hear people actually using slides! I'd love to see your slide deck (if you can share). Maybe I can have a folder of actual presentations given with slides in this repo so people can explore cool presentations (and use them as integration tests 😄)

**Pull request merged**